demisto_sdk/commands/create_artifacts/content_creator.py

```diff
@@ -169,17 +170,23 @@ def copy_dir_yml(self, dir_path, bundle):
         scan_files, _ = get_yml_paths_in_dir(dir_path, error_msg='')
         content_files = 0
         dir_name = os.path.basename(dir_path)
-        copy_func = self.copy_playbook_yml if dir_name in ['Playbooks', 'TestPlaybooks'] else self.copy_content_yml
         for path in scan_files:
             if len(os.path.basename(path)) >= self.file_name_max_size:
                 self.long_file_names.append(path)
 
             with open(path, 'r') as file_:
                 yml_info = yaml.safe_load(file_)
-
             ver = yml_info.get('fromversion', '0')
             print(f' - processing: {ver} ({path})')
-            copy_func(path, os.path.join(bundle, os.path.basename(path)), yml_info)
+
+            if dir_name in ['Playbooks', 'TestPlaybooks']:
+                self.copy_playbook_yml(path, os.path.join(bundle, os.path.basename(path)))
```

Comment:
`TestPlaybooks` folder can contain scripts, make sure this is handled correctly

Reply:
You don't want to get the scripts in the Testplaybook folder to the content bundle afaik.
what do you want to handle aside from that?

Reply:
those scripts should go to the test_bundle as they are being used by the test playbooks

Reply:
@yaakovi tended to it. first added the test case and fixed accordingly. hopefully I did not miss an edge case.